.github/workflows/angular-ci.yml

```
@@ -0,0 +1,84 @@
+name: Angular CI Workflow Template
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        description: 'Node.js version to use'
+        required: false
+        type: string
+        default: '24'
+      working-directory:
+        description: 'Directory containing Angular projects'
+        required: false
+        type: string
+        default: './gui'
+      checks:
+        description: 'List of checks to run (build, lint, test)'
+        required: false
+        type: string
+        default: '["build", "lint"]'
```

I feel like it isn't clear where the checks come from. I would propose renaming this to commands, since that is more accurate. Then specify where they come from in the description. For context, these come from the package.json file in our angular projects.


```suggestion
      commands:
        description: 'List of `npm run` commands to execute (build, lint, test). These come from the "scripts" section in your project's package.json
        required: false
        type: string
        default: '["build", "lint"]'
```

---

Ok, I believe this should be fixed now.